lib/auth/auth.go

```diff
@@ -1415,6 +1418,12 @@ func (a *Server) runPeriodicOperations() {
 			FirstDuration: retryutils.FullJitter(time.Minute),
 			Jitter:        retryutils.SeventhJitter,
 		},
+		interval.SubInterval[periodicIntervalKey]{
+			Key:           accessListReminderNotificationsKey,
+			Duration:      8 * time.Hour,
```

Comment:
To test locally, just replace this with `1 * time.Minute` so that the periodic runs often.